include/riak_kv_vnode.hrl

```diff
@@ -7,6 +7,18 @@
           start_time :: non_neg_integer(),
           options :: list()}).
 
+-record(riak_kv_w1c_put_req_v1, {
+    bkey :: {binary(),binary()},
+    encoded_val :: binary(),
```

Comment:
Would recommend `encoded_obj` instead to make sure people don't think it's the value that should be wrapped in a riak object at the vnode.